include/LightGBM/config.h

```diff
@@ -619,24 +621,27 @@ struct Config {
   // desc = enabling this will discretize (quantize) the gradients and hessians into bins of ``num_grad_quant_bins``
   // desc = with quantized training, most arithmetics in the training process will be integer operations
   // desc = gradient quantization can accelerate training, with little accuracy drop in most cases
-  // desc = **Note**: can be used only with ``device_type = cpu`` and ``device_type=cuda``
+  // desc = **Note**: works only with ``cpu`` and ``cuda`` device type
```

Comment:
"can be used only ..." for something that is not supposed to work in other environments at all, "Note: works only ..." for something that is supposed to work in other environments but currently doesn't work (not implemented yet).